packages/lb-annotation/src/core/pointCloud/index.ts

```diff
@@ -1203,6 +1315,25 @@ export class PointCloud {
     return { newBoxParams };
   }
 
+  public getNewSphereByBackUpdate(
+    offsetCenterPoint: { x: number; y: number; z: number }, // Just use x now.
+    offsetWidth: number,
+    offsetDepth: number,
+    selectedPointCloudSphere: IPointCloudSphere,
+  ) {
+    const Rz = new THREE.Matrix4().makeRotationZ(0);
+    const offsetVector = new THREE.Vector3(0, -offsetCenterPoint.x, 0).applyMatrix4(Rz);
```

Comment:
Delete `applyMatrix4(Rz);`. Don't need to use it

Reply:
Seems like the whole function was not used, i will delete it all.